src/plugins/unified_field_list/public/components/field_popover/field_popover_header.tsx

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import {
+  EuiButtonIcon,
+  EuiButtonIconProps,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPopoverProps,
+  EuiToolTip,
+  EuiTitle,
+} from '@elastic/eui';
+import { css } from '@emotion/react';
+import { i18n } from '@kbn/i18n';
+import type { DataViewField } from '@kbn/data-views-plugin/common';
+import type { AddFieldFilterHandler } from '../../types';
+
+const titleStyles = css`
+  text-transform: none;
+`;
```

Comment:
Out of curiosity, is this style necessary? I'm thinking this is probably a straggler override style from when the previous EUI theme was in use and forced uppercase on all popover headers. However, the current theme no longer does this. If no longer necessary, I imagine it can be removed.

Reply:
Removed styles, thanks @MichaelMarcialis !